network.go

```diff
@@ -61,7 +61,7 @@ type NetworkController interface {
 	Networks() []Network
 
 	// WalkNetworks uses the provided function to walk the Network(s) managed by this controller.
-	WalkNetworks(walker NetworkWalker)
+	WalkNetworks(walker NetworkWalker) bool
```

Comment:
This is another thing we aimed to avoid: Not to add to the interface use case specific requirements.
Unless the client really cannot get that information.

We would add today a bool return for the query of single Network use case. Tomorrow somebody comes with a use case where he wants a list of matching Network and require us to let him know if the Network list is empty.

I feel we should instead focus on finding what a client cannot retrieve based on today WalkNetworks() definition.